larray/core/ufuncs.py

```diff
@@ -302,3 +303,36 @@ def broadcastify(func):
      a0   6   2   5   0
      a1   5   2   3   0
 """
+
+
+def isscalar(element: Any) -> bool:
+    """
+    Returns `True` if the type of element is a scalar type.
+
+    Parameters
+    ----------
+    element: any
+        Input argument, can be of any type and shape.
+
+    Returns
+    -------
+    bool
+        `True` if `element` is a scalar type, `False` if it is not.
+
+    Examples
+    --------
+    >>> isscalar(3.1)
+    True
+    >>> isscalar(np.array(3.1))
```

Comment:
I wouldn't show this in the doctest (too obscure) for our users. On the other hand, a test with an **larray** scalar array could be interesting mostly to avoid having to add the information about that corner case to the description or the "Returns" paragraph **but** I would move it much lower in the list, because it is a corner case.